package.json

```diff
@@ -91,6 +91,7 @@
         "@typescript-eslint/eslint-plugin": "^7.0.0",
         "@typescript-eslint/parser": "^7.0.0",
         "babel-jest": "^29.0.0",
+        "babel-plugin-search-and-replace": "^1.1.1",
```

Comment:
I'd expect lockfile changes if we're adding a dependency.

Reply:
Quite right, added now.